lib/Service/LabelService.php

```diff
@@ -166,6 +167,7 @@ public function update($id, $title, $color) {
 		$label = $this->find($id);
 		//$this->checkDuplicateTitle($label->getBoardId(), $title);
 		
+		$boardLabels = array();
```

Comment:
you will need to create a mock in the tests for the findAll method on labelMapper, so that this either returns an empty array or better (to properly test the code) contain an actual list of labels that will then be used in the foreach loop during the test.